src/views/_base.njk

```diff
@@ -106,7 +106,7 @@
     meta: {
       items: [
         {
-          href: pathPrefix + '/privacy-policy',
+          href: "https://licensing.nature.scot/cross-service-pages/privacy-policy",
```

Comment:
I _think_ you can just use `href: "/cross-service-pages/privacy-policy",` and the host prefix should be correctly set - if you push up the change I can double check on the dev environment.